Allow passing -format fn for labels

mortenschioler opened the pull request:
I wanted to make my dep smaller by truncating the namespace qualifier
for the project, so I added this "hack". I am aware that this change
opens up for code injection, and if that is unacceptable, that's completely
OK -- go ahead and close this PR. Otherwise, I'll also take the time to
document this in the README.

mortenschioler committed:
Allow passing -format fn for labels

I wanted to make my dep smaller by truncating the namespace qualifier
for the project, so I added this "hack". I am aware that this change
opens up for code injection, and if that is unacceptable, that's completely
OK -- go ahead and close this PR. Otherwise, I'll also take the time to
document this in the README.

hilverd commented:
Thanks -- yes, could you please document this in the README before I merge it in?

mortenschioler committed:
Restructure and document `-format` in README

Group documentation of CLI options in subsection to
the new section `### Command-Line Options`.

mortenschioler commented:
> Thanks -- yes, could you please document this in the README before I merge it in?

Done in 2427a3e. This is my personal GitHub account BTW, hope that's not too confusing.

hilverd commented:
Thanks!
